openhands/core/config/condenser_config.py

```diff
@@ -0,0 +1,38 @@
+from typing import Literal
+
+from pydantic import BaseModel, Field
+
+from openhands.core.config.llm_config import LLMConfig
+
+
+class CondenserConfig(BaseModel):
```

Comment:
Just a small detail, you may want to check this bit, the other config classes are dataclasses instead of BaseModel. Some bunch of utility code works with them as dataclasses and assumes they all are, I think - stuff like ensure that their values are loaded correctly etc, e.g. https://github.com/All-Hands-AI/OpenHands/blob/f0ca2239f3e5010c379077af9bb33a34aa0bd067/openhands/core/config/utils.py#L42

Reply:
I'll double-check. Easy enough to change to data classes if needed!